### DISCUSSION TITLE
3d print case - Ender 3 v2

### DISCUSSION THREAD

**Discussion by chitown88** [riffnshred/nhl-led-scoreboard#382] [General]
I've been trying to print the case following the links from the tutorials. Brand new to this, using Ender 3 v2. Does anyone know the settings to use? Or have the gcode for that? I've been trying and failing at this as I don't know exactly how to set it up.

**Comment by riffnshred** [comment]
which case exactly are you trying to print. for the P3, P4 or the P5?

**Reply by chitown88** [reply]
P5

**Reply by riffnshred** [reply]
I never printed that case and I don't have an ender 3, but what kind of failure are you having?

**Reply by riffnshred** [reply]
are you new to 3d printing?

**Reply by chitown88** [reply]
Yes brand new. I think I sort of figured it out. I had to have the support turned on. And then I also rotated it so it printed it laying down as opposed to standing up. 

New issue is I have is I have the bed levelled. Base layer print flawless in sone parts, then others just randomly "stringy". Is this sort of a normal thing in that as there may be some slight dips and what not in the "flat" bed? Meaning just very minor flaws in the bed could cause this?

**Reply by riffnshred** [reply]
Now it all make sense. You are not asking at the right place for that stuff because 3d printing is something else but here's what I can tell you.  There is a learning curve that comes with 3d printing and you are going to waste a lot of plastic and time trying to print that case as you first thing to print. you are sort of trying to learn to drive but with a semitruck as your first car. 

The printer you have is also one of the cheapest one and you are going to have to deal with a bunch of problems at first until you get to that point where everything is properly setup and you just have to maintain that afterwards. I know that Ender 3 are praised online, but you have a lot of learning and work to do to get it to print properly. 

I recommend starting printing other smaller stuff until you get a hang of how 3d printing works (proper slicing for what you are trying to achieve)  and more specifically, how to get your 3d printer setup properly.

**Reply by chitown88** [reply]
Thanks for replying. I'm quickly learning it's not a simple, "download this model. Click print"   procedure, and I'm at the stage of "I don't know, what I don't know". 

I'll definitely take your advice and hone in those skills before diving into the complex prints. Thanks for your time.

**Comment by riffnshred** [comment]
No worries. Its a hobby on its own and I hope you enjoy it as much as I do.

Cheers
